### DISCUSSION TITLE
ITMS-91061: Missing privacy manifest

### DISCUSSION THREAD

**Discussion by ShubhaPradhan** [firebase/flutterfire#13270] [Q&A]
Your app includes “Frameworks/FirebaseCore.framework/FirebaseCore”, which includes FirebaseCore, an SDK that was identified in the documentation as a privacy-impacting third-party SDK. Starting November 12, 2024, if a new app includes a privacy-impacting SDK, or an app update adds a new privacy-impacting SDK, the SDK must include a privacy manifest file. Please contact the provider of the SDK that includes this file to get an updated SDK version with a privacy manifest. For more details about this policy, including a list of SDKs that are required to include signatures and manifests, visit:
[https://developer.apple.com/support/third-party-SDK-requirements](url)

<img width="688" alt="Screenshot 2024-09-04 at 12 44 33" src="https://github.com/user-attachments/assets/9ae08325-ec49-447f-bf61-c89cf174d104">

**Comment by mohammadprgrm20123** [comment]
I have the same problem

**Comment by ravindraStackves** [comment]
i have same

**Comment by tmwebs** [comment]
i have same

**Comment by gitstc** [comment]
Any updates on this?

**Comment by JonasRech** [comment]
any update? same here

**Comment by deiv061** [comment]
Any updates?

**Comment by Nandulucky** [comment]
I have the same problem
